config/norns-crone.service

```diff
@@ -1,5 +1,6 @@
 [Unit]
-Wants=norns-jack.service
+After=norns-jack.service
+Requires=norns-jack.service
```

Comment:
Can the new standalone crone handle JACK not being up when it starts or JACK going away and coming back again when it's running?

Reply:
jack needs to be running when crone starts, or crone will fail. if jack quits while crone is running, it will not reconnect.

RESET in the system menu restarts the services successfully.

Reply:
OK, in that case these should be the correct type of dependencies :)